tasks/section_4/cis_4.3.x.yml

```diff
@@ -17,9 +17,9 @@
 - name: "4.3.2 | PATCH | Ensure sudo commands use pty"
   ansible.builtin.lineinfile:
       path: /etc/sudoers
-      regexp: '^Defaults        use_'
-      line: 'Defaults        use_pty'
-      insertafter: '^Defaults'
+      regexp: '^\s*Defaults\s+use_pty\s*$'
```

Comment:
Look for a wider range of compatible regexps...in particular, I had a file that mirrored the format of the lines in the distribution file and used a tab to separate. Instead of calling it correct or replacing the line, it added a new line.